deployment/enterprise_sso/enterprise_aws_sso_stack.py

```diff
@@ -484,3 +497,45 @@ def _create_lambda_role(
             )
             lambda_role.add_to_principal_policy(sts_policy)
         return lambda_role
+
+
+@jsii.implements(ILocalBundling)
+class LocalBundler:
+    """This allows packaging lambda functions without the use of Docker"""
+
+    def __init__(self, source_root):
+        self.source_root = source_root
+
+    def try_bundle(self, output_dir: str, options: BundlingOptions) -> bool:
+        try:
+            subprocess.check_call([sys.executable, "-m", "pip", "--version"])
+        except:
+            return False
+
+        python_output_dir = str(Path(output_dir, "python"))
+        subprocess.check_call(
+            [
+                sys.executable,
+                "-m",
+                "pip",
+                "--no-cache-dir",
+                "install",
+                "-r",
+                str(Path(self.source_root, "requirements.txt")),
+                "-t",
+                python_output_dir,
+            ]
+        )
+
+        def copytree(src: str, dst: str, symlinks=False, ignore=None):
+            for item in os.listdir(src):
+                source_item = os.path.join(src, item)
+                destination_item = os.path.join(dst, item)
+                if os.path.isdir(source_item):
+                    shutil.copytree(source_item, destination_item, symlinks, ignore)
+                else:
+                    shutil.copy2(source_item, destination_item)
+
+        copytree(self.source_root, python_output_dir)
```

Comment:
Is the creation of the folder for `sso`, `orgz`, ... no longer needed as you `cp` the whole folder (see line 305), or do we still need this?

Reply:
the command on line 305 is run in docker. the LocalBundler runs its own logic